### DISCUSSION TITLE
Need Help with teams membership REST API

### DISCUSSION THREAD

**Discussion by RInkeshIdx9** [community/community#55824] [Apps, API and Webhooks]
### Select Topic Area

API related question

### Body

Hey guys,
I am working on integrating GitHub with my application and capture organizations memberships, teams and team memberships but when I tried [List team membership](https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28#list-team-members) on Postman I found that I am getting all the members for a team including child team memberships.
Is their any API that I was not able to found to get direct assigned membership for a team and not the child team.
Also I have to remove the child team from the parent team through API.

Any help would be appreciated, Thanks in advance.

**Comment by byrneh** [comment]
This may help.
Try the GrapghQL API querty Orgaization, teams
You can [Delete](https://docs.github.com/en/rest/teams/teams?apiVersion=2022-11-28#delete-a-team) a team through the REST API. Deleting a parent team will delete all of its child teams as well.

**Reply by RInkeshIdx9** [reply]
Thanks @byrneh for answer, I tried GraphQl as well this is also giving me all the members including child team members.
`query {
  organization(login: "your_organization") {
    team(slug: "your_team_slug") {
      members(first: 100) {
        nodes {
          login
        }
      }
    }
  }
}
`
